docs/source/raft_bft_migration.md

```diff
@@ -0,0 +1,152 @@
+# Migrating from Raft to BFT
+
+Nodes at v3.0.0 or higher allow for users who want to transition channels from using Raft-based ordering services to BFT-based ordering services.
+This can be accomplished through a series of configuration update transactions on each channel in the network.
+
+This tutorial will describe the migration process at a high level, calling out specific details where necessary.
+
+
+## Assumptions and considerations
+Before attempting migration, take the following into account:
+
+1. This process is solely for migration from Raft to BFT. Migrating between any other orderer consensus types is not supported.
+2. Migration is one way. Once the ordering service is migrated to BFT, and starts committing transactions, it is not possible to go back to Raft.
+3. Because the ordering nodes must go down and be brought back up, downtime must be allowed during the migration.
+4. Recovering from a botched migration is possible only if a backup is taken at the point in migration prescribed later in this document. If you do not take a backup, and migration fails, you will not be able to recover your previous state.
+5. All channels must be migrated during the same maintenance window. It is not possible to migrate only some channels before resuming operations.
```

Comment:
Here "All channels" means all the channels managed by the targeted OSN, considering there's no system channel in v3.0.0.